process faults {4: ({2, 5}, 653088039)}
network faults {2: [{0, 4}, {6, 1}, {2}, {5, 3}], 6: [{0}, {1, 3, 6}, {4, 2}, {5}]}
{8: ("10F755B2AA16114821324D438BF052FEAF43C2BAF711A74815FB16E47DEB191A", {5, 3, 1, 6, 0, 2, 4}), 7: ("63AD5055C720A3EB4422A6F36CE335C2BA421788FBABE56ECA7A90073606F084", {5, 6, 2, 0, 1, 3, 4}), 9: ("06B0BBE5ACDFFC64DD755A02DD3F59764A03BF825A2BABA66BD5DBEF5EBC9033", {5, 2, 3, 1, 4, 6, 0}), 5: ("914D68A24C0930A67C3CCC0184C316FDEB4D9CD5D8AC88137372EB360D592A7A", {5, 6, 2, 3, 1, 0, 4}), 10: ("C9BB4D523A069FFE0C1E2C7BEC384086965D1CAF4A82546CDCCC4F41D355C154", {0, 2, 6, 5, 1, 4, 3}), 6: ("78B6B925807EEC43D7CBFC00BD0CD773EEB8FB06CBEE45710D25FEEF2345AF42", {5, 4, 2, 0, 1, 3, 6})}
7
reason: all committed
done!
